kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-09T00:00:00Z
+---
+
+All client-side requests made to the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) are [gRPC requests](https://grpc.io/docs/what-is-grpc/core-concepts/#deadlines).
```

Comment:
Would say that that they go through the temporal frontend service...then tie it with my comment (second below)